main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -32,13 +33,18 @@ const { creatorInvitation } =
   await E(zoe).startInstance(installation, issuerKeywordRecord);
 ```
 
-Then she escrows her offer with Zoe. When she escrows, she passes in two
-things, the actual ERTP payments that are part of her offer, and a
-`proposal`. The proposal will be used by Zoe to protect Alice from the
-smart contract and other participants. The proposal has three parts:
-`give` and `want`, which are used for enforcing offer safety, and `exit`,
-which is used to enforce exit safety. In this case, Alice's exit rule is
-`onDemand`, meaning that she can exit at any time.
+Then Alice escrows her offer with Zoe. She passes in two
+things; the actual ERTP payments of her offer, and a
+`proposal`. Zoe uses the proposal to protect Alice from the
+smart contract (which may have been written by someone else) 
+and other participants. 
+
+A proposal has three parts:
+- `give` and `want` enforce offer safety.
+- `exit` enforces exit safety. 
```

Comment:
give, want, and exit don't enforce safety. Zoe uses them as spec on which to base its safety calculations.